pywinauto/controls/uia_controls.py

```diff
@@ -1302,6 +1308,111 @@ def check_button(self, button_identifier, make_checked, exact=True):
+    def _activate(self, item, is_last):
+        """Activate the specified item"""
+        if not item.is_active():
+            item.set_focus()
+        try:
+            item.expand()
+        except NoPatternInterfaceError:
+            if not is_last:
+                item.select()
+
+    # -----------------------------------------------------------
+    def _sub_item_by_text(self, menu, name, exact, is_last):
+        """Find a sub-item by the specified text"""
+        sub_item = None
+        items = menu.items()
+        if items:
+            if exact:
+                for i in items:
+                    if name == i.window_text():
+                        sub_item = i
+                        break
+            else:
+                texts = []
+                for i in items:
+                    texts.append(i.window_text())
+                sub_item = findbestmatch.find_best_match(name, texts, items)
+
+        self._activate(sub_item, is_last)
+
+        return sub_item
+
+    # -----------------------------------------------------------
+    def _sub_item_by_idx(self, menu, idx, is_last):
+        """Find a sub-item by the specified index"""
+        sub_item = None
+        items = menu.items()
+        if items:
+            sub_item = items[idx]
+        self._activate(sub_item, is_last)
+        return sub_item
+
+    # ----------------------------------------------------------------
+    items = buttons
+
+    def item_by_path(self, path, exact=False):
+        """
+        Walk the items in this toolbar to find the item specified by a path
+
+        The path is specified by a list of items separated by '->'. Each item
+        can be either a string (can include spaces) e.g. "Save As" or a zero
+        based index of the item to return prefaced by # e.g. #1.
+
+        These can be mixed as necessary. For example:
+            - "#0->Save As",
+            - "Tools->#0->Configure"
+
+        * **path** - Path to the specified item. **Required**.
+        * **exact** - If false, text matching will use a 'best match' fuzzy algorithm. If true, will try to find the
+                      item with the given name. (Default False). **Optional**
+        """
+        toolbar_items = [p.strip() for p in path.split("->")]
+        items_cnt = len(toolbar_items)
+        if items_cnt == 0:
+            raise IndexError()
+        for item in toolbar_items:
+            if not item:
+                raise IndexError("Empty item name between '->' separators")
+
+        def next_level_menu(parent_menu, item_name, is_last):
+            if item_name.startswith("#"):
+                return self._sub_item_by_idx(parent_menu, int(item_name[1:]), is_last)
+            else:
+                return self._sub_item_by_text(parent_menu, item_name, exact, is_last)
+
+        try:
+            if items_cnt == 1:
+                menu = next_level_menu(self, toolbar_items[0], items_cnt == 1)
+                return menu
+            else:
```

Comment:
`else:` is not necessay after `return` statement (Codacy warning). Remove else and de-indent the below code block.